cloud/ansible/copy-src.yaml

```diff
@@ -0,0 +1,18 @@
+- name: Copy sources
+  hosts: public-machine
+  become: no
+  tasks:
+    - name: copy frontend src
+      ansible.posix.synchronize:
+        src: ../../frontend
+        dest: /home/azureuser/
+
```

Comment:
czy to nie powinno byc zaciagane z gita po prostu?

Reply:
nooo nie wiem w sumie, z jednej strony tak, z drugiej strony to pozwala na testowanie lepsze bo nie trzeba wypychać zmian których się nie jest pewnym, zastanowie się jeszcze nad tym i zobaczę czy da się bezboleśnie repo sklonować bez podawania secretów

Reply:
repo jest publiczne, wiec powinno byc elegancko, nie widze chyba opcji gdzie chcialbys cos testowac co jest niezacommitowane ale deployowac to na serwerze tbh

Reply:
aight, nie pamiętałem że repo jest publiczne. W takim razie tak zrobie